### ISSUE TITLE
br issue

### ISSUE THREAD

**Issue by idansh** [im4aLL/easyeditor#2]
When I load the editor everything works fine. When I inject the HTML of the editor and initialize it, when clicking enter, it put an extra <br>. So the third time, it put three <br> instead of one.

I don't know why it works like that, because I can't find anything in the local storage or something that remembers the last input. I override the html and the textarea.

After debugging, when I press ENTER, the line: 'easyEditor.prototype.handleKeypress'  runs twice and goes to the line that prints <br><br>.

**Comment by im4aLL** [comment]
Yes, I noticed that also when allowing multiple editors in same page. Please do pull the latest js and I slight changed the calling system. Let me know if the issue still not solved yet from latest JS.

**Comment by im4aLL** [comment]
Also please check this URL - http://habibhadi.com/lab/easyeditor/multiple-easyeditor.html

**Comment by idansh** [comment]
For some reason it now throws (after I downloaded the new file) the following error in line 37:
"Uncaught TypeError: Cannot read property 'toLowerCase' of undefined".

I don't use bootstrap by the way. Any suggestions.

Until it resolved, I currently still use this line to prevent the further delegation of events, clearing them up.

``` javascript
  $(document).on('closing', '.remodal', function (e) {
        $('html').off()

    });
```

**Comment by im4aLL** [comment]
Are you getting error in default initial or issue is occurring on any button click? Can I see any sample code of yours so that I can replicate the issue in my end?

**Comment by idansh** [comment]
``` javascript
            var easyEditor = new EasyEditor({
                element: '#user-comment-in-popup',
                buttons: ['bold', 'italic', 'quote','youtube', 'x'],
                buttonsHtml: {
                    'bold': '<i class="fa fa-bold"></i>',
                    'italic': '<i class="fa fa-italic"></i>',
                    'quote': '<i class="fa fa-quote-left"></i>',
                    'insert-youtube-video': '<i class="fa fa-youtube"></i>',
                    'remove-formatting': '<i class="fa fa-ban"></i>'
                }
            });
```

On init it happens.

**Comment by im4aLL** [comment]
Yes, you are still using previous calling method. Please see the new calling method. And it should be like this 

new EasyEditor('#user-comment-in-popup', { options: options}); 

Sorry using mobile currently.

**Comment by idansh** [comment]
Works flawlessly. Just adding, I went through a more than 50 WYSIWYG plugins, chose yours for its simplicity. Thanks!

**Comment by im4aLL** [comment]
Yeah man! I went through many rich text editor as well. But found those are overhead and overcoded. Though I built it recently, but it serves the purpose I guess. Some of them doesn't even allow to add simple button without payment. I believe those are good as well running from years. 

My main goal is to make it simple and with minimal css so that anybody can integrate with any site with just couples of css tweaking. I made every possible options configurarable. People can add tons of buttons even any file manager with their own. 

Thanks for such great feedback it will keep my inspiration ahead.

Thanks
Hadi
